app/src/main/res/values/styles.xml

```diff
@@ -378,25 +378,28 @@
     <style name="HeaderTextStyle" parent="TextAppearance.MaterialComponents.Subtitle1">
         <item name="android:textSize">18sp</item>
         <item name="android:textColor">?primaryText</item>
-        <item name="fontFamily">@font/metropolis_semibold</item>
+        <item name="fontWeight">@integer/font_weight_medium</item>
     </style>
 
     <style name="Header20TextStyle" parent="TextAppearance.MaterialComponents.Body1">
         <item name="android:textColor">?primaryText</item>
         <item name="android:textSize">20sp</item>
-        <item name="fontFamily">@font/metropolis_semibold</item>
+        <item name="fontWeight">@integer/font_weight_medium</item>
     </style>
 
     <style name="Header16TextStyle" parent="TextAppearance.MaterialComponents.Body1">
         <item name="android:textColor">?primaryText</item>
         <item name="android:textSize">16sp</item>
-        <item name="fontFamily">@font/metropolis_semibold</item>
+        <item name="android:textFontWeight" tools:ignore="NewApi">@integer/font_weight_medium</item>
```

Comment:
@gabrielluong For reference, this is the workaround mentioned in [this](https://github.com/mozilla-mobile/fenix/pull/22844#issuecomment-1030246430) comment, that produces the results in the attached screenshots.

Reply:
Seeing the "NewApi" - I think we should just make sure we test for Android 6 since that was the last problematic version I had to deal with.